README.md

```diff
@@ -1 +1,73 @@
-# Hello-World-2
\ No newline at end of file
+# Hello-Microvese
+
+- First Html project in Microverse.
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Screen Shot
+
+![screenshot](./app_screenshot.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- Html
+- Css
+- Vscode
+- linters
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
```

Comment:
- [ ] This demo link is not working correctly. Please use [Github Pages](https://pages.github.com/) to host your demo project and provide the link accordingly at this part of the ReadMe.